packages/vite/src/node/config.ts

```diff
@@ -547,7 +552,7 @@ async function loadConfigFromBundledFile(
   return config
 }
 
-export function loadEnv(mode: string, root: string, prefix = 'VITE_') {
+export function loadEnv(mode: string, root: string, unsafeEnv: boolean) {
```

Comment:
The changelog of https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md#features-34 tells that `loadEnv` is explicitly exposed to public API.
So changing the third parameter is a breaking change :warning:

Reply:
```ts
export function loadEnv(mode: string, root: string, prefix: `VITE_${string}` | boolean | undefined = 'VITE_') {
  if (typeof prefix === 'boolean') ...
```

This will prevent breaking change